stable/datadog/templates/daemonset.yaml

```diff
@@ -25,13 +28,22 @@ spec:
 {{ toYaml .Values.resources | indent 12 }}
         ports:
         - containerPort: 8125
+          {{- if .Values.daemonset.useHostPort }}
+          hostPort: 8125
+          {{- end }}
           name: dogstatsdport
           protocol: UDP
         {{- if .Values.datadog.apmEnabled }}
         - containerPort: 8126
+          {{- if .Values.daemonset.useHostPort }}
+          hostPort: 8126
+          {{- end }}
           name: traceport
           protocol: TCP
         - containerPort: 7777
+          {{- if .Values.daemonset.useHostPort }}
```

Comment:
this is the legacy port, we don't need to support it for new options and we'll actually remove it soon